sycl/include/CL/sycl/INTEL/esimd/detail/esimd_intrin.hpp

```diff
@@ -14,11 +14,12 @@
 #include <CL/sycl/INTEL/esimd/detail/esimd_types.hpp>
 #include <CL/sycl/INTEL/esimd/detail/esimd_util.hpp>
 #include <CL/sycl/INTEL/esimd/esimd_enum.hpp>
-#include <CL/sycl/detail/accessor_impl.hpp>
 
 #include <assert.h>
 #include <cstdint>
 
+#define __SIGD sycl::INTEL::gpu::detail
```

Comment:
I'm a little concerned about the name here. Maybe something like this would be better?
```
namespace sigd = sycl::INTEL::gpu::detail;
```
Similar to what we have for `csd` namespace.

Reply:
the problem with such short names in headers in the global namespace is that they can conflict with other code. {{__}} factors out user code.

Reply:
Yes, that's true. Do you know if we have project guidelines for such cases?

Reply:
The only guideline I know is the llvm coding style.

Reply:
The reason why I did not use `namespace __SIGD = sycl::INTEL::gpu::detail;` instead is because it is in the global scope and can't be undef'ed, unlike a macro. Which means all subsequent headers and user code will see that declaration. Here I just #undef once no longer needed.